test/ipynb/mpl/graph/test_graph_matplotlib_drawer.py

```diff
@@ -157,6 +164,28 @@ def test_plot_histogram(self):
 
         self.graph_count_drawer(counts, filename="histogram.png")
 
+    def test_plot_gate_map(self):
+        """for testing the plot_gate_map"""
+        # getting the mock backend from FakeProvider
+
+        backend = FakeBelem()
+
+        self.graph_plot_gate_map(backend=backend, filename="gate_map.png")
+
+    def test_plot_coupling_map(self):
+        """for testing the plot_coupling_map"""
+
+        num_qubits = 5
+        qubit_coordinates = [[1, 0], [0, 1], [1, 1], [1, 2], [2, 1]]
+        coupling_map = [[0, 1], [1, 0], [1, 2], [1, 3], [2, 1], [3, 1], [3, 4], [4, 3]]
```

Comment:
I think you can remove some of these edges (e.g. [0,1] and [1,0] are just redrawing the same line as far as I'm aware)

Reply:
Removed the duplicate edges. 👍